.agent/skills/ss14-netcode/SKILL.md

```
@@ -0,0 +1,422 @@
+---
+name: SS14 Netcode Architecture
+description: Architecture guide for networking in Space Station 14 — Lidgren integration, NetManager abstraction, message system, game state synchronization, PVS, network events, and component networking
+---
+
+# SS14 network architecture
+
+## Stack overview
+
+The SS14 networking stack consists of several layers of abstraction, from low-level transport to game state synchronization logic:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)